### DISCUSSION TITLE
Exposed Adguard Home interface is blacklisted by google, even with HTTPS and valid certificates

### DISCUSSION THREAD

**Discussion by vlebourl** [AdguardTeam/AdGuardHome#5283] [Q&A]
Hi, I juste installed Adguard Home on a virtual machine hosted on my Freebox Delta server (French ISP). I'm trying to expose the web interface through https but every time I expose it publicly I get blacklisted by google with the following red screen.

I've tried reverse proxy or direct port redirection, it doesn't change anything... Any idea what might be the cause? 

Looking the address up on https://sitecheck.sucuri.net, only google is blacklisting it...
![image](https://user-images.githubusercontent.com/5701372/208403987-971f8f19-85f2-41dd-abde-58e3a2162a34.png)

Cheers!
![image](https://user-images.githubusercontent.com/5701372/208403811-6d5d9995-9cfc-44b8-a682-5ad95d69a7f7.png)

![image](https://user-images.githubusercontent.com/5701372/208403583-7be46ea5-0c91-45d8-8f26-ba33f7fc02fd.png)

**Comment by Disqu3-mirOir-qt** [comment]
it's not a problem related to AdguardHome

you need to contact google or report a false positive

it may be that your domain name provider is not secure or trusted

**Comment by ammnt** [comment]
@vlebourl, use this URL to report blocking error:
https://safebrowsing.google.com/safebrowsing/report_error/?hl=en
